SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -751,13 +751,8 @@ int64 USpatialActorChannel::ReplicateActor()
 	{
 		FOutBunch DummyOutBunch;
 
-		// Actor::ReplicateSubobjects is overridable and enables the Actor to replicate any subobjects directly, via a
-		// call back into SpatialActorChannel::ReplicateSubobject, as well as issues a call to UActorComponent::ReplicateSubobjects
-		// on any of its replicating actor components. This allows the component to replicate any of its subobjects directly via
-		// the same SpatialActorChannel::ReplicateSubobject.
-		Actor->ReplicateSubobjects(this, &DummyOutBunch, &RepFlags);
-
-		// Look for deleted subobjects
+		// Look for deleted subobjects before trying to replicate existing ones; this will free the removed Unreal components' bound Spatial
+		// components for reuse.
 		for (auto RepComp = ReplicationMap.CreateIterator(); RepComp; ++RepComp)
```

Comment:
summoning @MCArth-improbable to have a look, the ReplicationMap is involved

Reply:
seems fine

Reply:
though tangentially neither us nor native never remove objectreplicators from the replicationmap on clients (apart from if the actor dies), which is kinda funny